utilities/openemr-cmd/openemr-cmd

```diff
@@ -229,50 +166,51 @@ which docker &>/dev/null
-	echo "  backup-snapshot              Create a backup snapshot"
-	echo "  restore-snapshot             Restore from a snapshot"
-	echo "  list-snapshots               To list the snapshots"
-	echo "  list-capsules                List the capsules"
-	echo "  get-capsule                  Copy the capsule from the docker to your host directory"
-	echo "  put-capsule                  Copy the capsule into the docker"
-	echo "  ensure-version               Upgrade OpenEMR from specified old version to current version"
-	echo "  enable-multisite             Turn on support for multisite"
-	echo "  disable-multisite            Turn off support for multisite"
-	echo "  encoding-collation           Change the database character set and collation"
+  echo "  -h, --help                         Show the commands usage"
+  echo "  -v, --version                      Show the openemr-cmd command version"
+  echo "  -s, shell                          Open a docker shell quickly"
+  echo "  up                                 Execute: docker-compose up -d"
+  echo "  down                               Execute: docker-compose down -v"
+  echo "  start                              Execute: docker-compose start"
+  echo "  stop                               Execute: docker-compose stop"
+  echo "  bt, build-themes                   Make changes to any files on your local file system"
+	echo "  pl, php-log                        To check PHP error logs"
+	echo "  dl, docker-log                     To check docker log"
+	echo "  dn, docker-names                   To check docker the running docker names"
+	echo "  pr, psr12-report                   To create a report of PSR12 code styling issues"
+	echo "  pf, psr12-fix                      To fix PSR12 code styling issues"
+	echo "  ltr, lint-themes-report            To create a report of theme styling issues"
+	echo "  ltf, lint-themes-fix               To fix theme styling issues"
+	echo "  pp, php-parserror                  To check PHP parsing errors"
+	echo "  ut, unit-test                      To run unit testing"
+	echo "  at, api-test                       To run api testing"
+	echo "  et, e2e-test                       To run e2e testing"
+	echo "  st, services-test                  To run services testing"
+	echo "  ft, fixtures-test                  To run fixtures testing"
+	echo "  vt, validators-test                To run validators testing"
+	echo "  ct, controllers-test               To run controllers testing"
+	echo "  ctt, common-test                   To run common testing"
+	echo "  cs, clean-sweep                    To run the entire dev tool suite"
+	echo "  cst, clean-sweep-tests             To run only all the automated tests"
+	echo "  dr, dev-reset                      To reset OpenEMR only"
+	echo "  dri, dev-reset-install             To reset and reinstall OpenEMR"
+	echo "  drid, dev-reset-install-demodata   To reset and reinstall OpenEMR with demo data"
```

Comment:
i like the sound if it:
```
drid
```
![tenor (4)](https://user-images.githubusercontent.com/278968/87869832-c7399980-c957-11ea-9264-12da74573a3d.gif)

kidding aside, though, you keep making devs likes easier. nice that you are working on the insane dev environment. i was planning to update those dockers to fully support devtools (the backup/restore/demo stuff doesn't work), so this is a good reason to do that. when that is up and running, then I would do the following when don't include a docker name:
see if following exist:
```
$(docker ps | grep _openemr_1 | cut -f 1 -d " ")
```
if not there, then see if following exists:
```
$(docker ps | grep _openemr-7-3-312_1 | cut -f 1 -d " ")
```
if not, then use the provided argument

Reply:
@bradymiller, haha, yep. I think openemr-cmd is already almost can cover easy env. But insane env,  I think this "_openemr-7-3-312_1" should gonna be an argument in insane dev.  And the openemr container version, it seems that will keep changing in the future. Not sure it needs another version, or can also cover insane env.

Reply:
the goal is all the devtools will work on insane environment just like easy dev (just need to do some trickery in the flex docker to support; i think i know what i need to do, hopefully get around to it over the next week or so)